ev3dev/auto.py

```diff
@@ -8,11 +8,17 @@ def current_platform():
         return 'ev3'
     elif machine == 'armv6l':
         return 'brickpi'
+    elif machine == 'armv7l':
```

Comment:
So you only attach BrickPi to RPi 1 and PiStorms to RPi 2? What about RPi 3 (`armv8...`)? Also, Beaglebone is `armv7l`, so it will be incorrectly detected as `pistorms`.

See ev3dev/ev3dev#412